### PR TITLE
Update check box label

### PR DESCRIPTION
**Pull request by Alverd04**
## molecule/checkboxField & field

**TASK**: [SCMI-59671](https://jira.scmspain.com/browse/SCMI-59671)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context
Updated components to work with a isBold prop that allows the developer to set label boldness without affecting parents components. 

### Screenshots - Animations
image

### CODE REVIEW

File: components/molecule/checkboxField/src/index.js [hunk 1]
```diff
@@ -15,13 +15,15 @@ const MoleculeCheckboxField = ({
   alertText,
   helpText,
   onChange,
+  isBoldLabel,
```
Comment (danivm) [inline components/molecule/checkboxField/src/index.js:18]: if this prop is not required it would be nice to have a default prop like `false`

**Review by danivm — APPROVED** [review]

**Comment by turolopezsanabria** [comment]
I suggest avoiding boolean props
Using options will prepare us for future use cases (imagine we need a lighter font-weight one day)

I rather suggest a prop named `fontWeight`, with the following options:

- regular (this one by default)
- bold

**Comment by turolopezsanabria** [comment]
Hello @Alverd04 🙋🏻‍♂️
Do you need help to progress with this PR?

**Comment by Alverd04** [comment]
> Hello @Alverd04 🙋🏻‍♂️
> Do you need help to progress with this PR?

Hi, this branch should be closed, we did it in another component. Sorry!!!!!!!!!

**Pull request closed without merging**